### DISCUSSION TITLE
Mini RPS, a reverse proxy based on axum

### DISCUSSION THREAD

**Discussion by marcodpt** [tokio-rs/axum#2279] [Show and tell]
I would like to present my work [minirps](https://github.com/marcodpt/minirps), it is a server with:

 - very fast single binary with no dependencies
 - static file server
 - CORS
 - HTTPS
 - reverse proxy with templates

I really liked the end result.

I would like to thank you very much for the work done on axum.

**Comment by jplatte** [comment]
Feel free to send a PR adding your project to `ECOSYSTEM.md` 🙂

**Reply by marcodpt** [reply]
@jplatte sorry for the late response!

I'm a lone developer and time is scarce.

I submitted a [pull request](https://github.com/tokio-rs/axum/pull/2861). 

It's very difficult to publicize my work, I really appreciate the openness given.
